server/src/controllers/employer_contact/addEmployerContact.js

```diff
@@ -0,0 +1,47 @@
+const logger = require("pino")();
+const EmployerContact = require("../../models/employer_contact.model");
+const Employer = require("../../models/employer.model");
+
+const addEmployerContactRequestHandler = async (req, res) => {
+  try {
+    const { name, email, job_type, phone_number, alt_phone_number, employer } =
+      req.body;
+    if (!name || !email || !phone_number) {
```

Comment:
I think it could potentially be better to rewrite this message by keeping track of which ones are missing, and then outputting the message as such (could make displaying the error on the frontend easier)
ex
`
```suggestion
    const errors = []
    if (!name) {
        errors.push("name");
    }
    if (!email) {
        errors.push("email");
    }
    if (!phone_number) {
        errors.push("phone number");
    }
   if (errors.length > 0) {
      message = errors.join(", ");
      return res.status(400).json({
          status: "fail",
          message: `The following fields were missing: ${message}`,
          data: null,
      });
   }
```
let me know what you think!